examples/advanced/fl_hub/config/site_a/hub_server.json

```diff
@@ -0,0 +1,23 @@
+{
+  "format_version": 2,
+  "workflows": [
+    {
+      "id": "controller",
+      "path": "nvflare.app_common.hub.hub_controller.HubController",
+      "args": {
+        "pipe_id": "pipe",
+        "task_wait_time": 60,
```

Comment:
You don't have to specify this. If not specified, the HubController will keep waiting until either a task is received or the peer is disconnected (based on heartbeat).